promql/quantile.go

```diff
@@ -370,18 +372,22 @@ func coalesceBuckets(buckets buckets) buckets {
 //
 // As a somewhat hacky solution until ingestion is atomic per scrape, we
```

Comment:
Actually, ingestion _is_ atomic per scrape now. But there are still other ways of encountering inconsistent buckets (data is already inconsistent at the target's side, histograms are ingested via the remote write receiver that Prometheus implements).

So this doc comment needs an update, otherwise future readers will be tempted to remove the whole thing.

Reply:
I've updated the doc comment, but not sure I understood the change correctly, please check.

Reply:
I think it's fine. Most important was that we don't state a reason that isn't true anymore. I think your explanation is even more than needed, but it also cannot harm to provide this context here.